packages/esm-patient-search-app/src/patient-search-result/patient-search-result.component.tsx

```diff
@@ -1,61 +1,87 @@
-import React from 'react';
-import dayjs from 'dayjs';
+import React, { useCallback, useMemo } from 'react';
 import styles from './patient-search-result.scss';
-import { interpolateString, ExtensionSlot, useConfig, navigate } from '@openmrs/esm-framework';
+import { ExtensionSlot, useConfig, interpolateString, navigate } from '@openmrs/esm-framework';
 import { SearchedPatient } from '../types/index';
 
+const RenderPatient: React.FC<RenderPatientProps> = ({ patient, onClickResult }) => {
```

Comment:
A component should probably never have "Render" in the name. Call `PatientSearchResult` or, better, just inline it in `PatientSearchResults`.

Reply:
@brandones Updated :)